src/main/java/nus/edu/rizzler/Rizzler.java

```diff
@@ -0,0 +1,97 @@
+package nus.edu.rizzler;
+
+import java.util.Scanner;
+import nus.edu.rizzler.exception.InvalidInputException;
+import nus.edu.rizzler.manager.TaskManager;
+import nus.edu.rizzler.ui.Emoji;
+
+public class Rizzler {
+    Emoji emoji = new Emoji();
```

Comment:
The emoji object seems to be created in every class. You can consider having a single instance instead.